api/src/controllers/cards.controller.ts

```diff
@@ -24,7 +24,11 @@ export class CardsController {
   // Controller to get all cards this will be used for the main page.
   public getCards = async (req: Request, res: Response, next: NextFunction) => {
     try {
-      const findAllCardsData: PokemonCard[] = await this.card.findAllCards();
+      // Initialize our page to 1 and our limit to 12 (12 cards per page)
+      const page = req.query.page || 1;
+      const limit = 12;
+
```

Comment:
May also want to add a small check here to see if the value of `page` is greater than `totalNumberOfCards / limit` (because that page would not exist). If `page` is greater than `totalNumberOfCards / limit`, then we can return an error code or something. 

I'm not sure if there is a way to find out the totalNumberOfCards in our system. If there is not, you can disregard this comment

Reply:
I made changes to the getCards function so the page won't be greater than the number of possible pages. For example, if page 2 is the final page and page 4 is passed the URL the getCards function now automatically just changes the value of the page to the final possible page in this example it would be 2. Not sure if this is best approach but it was what I thought of so the user does not go past the final page.

Reply:
That is smart, I like that approach. Maybe we should also implement something in the opposite direction too (To check if the inputted page is less than 1 (negative page). Then, we would automatically change the page value to the first page.) I'm going to approve this pull request, but you can add that change whenever in the future